themes/classic/templates/checkout/_partials/cart-detailed-totals.tpl

```diff
@@ -47,29 +47,9 @@
     {/foreach}
   </div>
 
-  <div class="card-block cart-summary-totals">
```

Comment:
Can we keep this class? It may be used by themes using classic as parent.

Reply:
we could expect something like

```tpl
   {block name='cart_summary_totals'} 
     <div class="card-block cart-summary-totals"> 
       {include file='checkout/_partials/cart-summary-totals.tpl' cart=$cart} 
     </div> 
   {/block} 
```

Reply:
This class is already used in `checkout/_partials/cart-summary-totals.tpl`

https://github.com/PrestaShop/PrestaShop/blob/develop/themes/classic/templates/checkout/_partials/cart-summary-totals.tpl#L25

Reply:
yes except you changed `card-block cart-summary-totals` with `card-block cart-summary-totals-container` so it actually changes your block class and potentialy css
I'm aware the class is also used in the loop..

but maybe you could use `.card-block.cart-summary-totals` as a selector or add an id `id=cart-summary-totals-container` or `id=cart-summary-totals-list` instead (unless the block is displayed several times in the page and you need to update all of them?)